_posts/blog/2017-04-21-introducing-uno-and-muchos.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Introducing Uno and Muchos"
+author: Mike Walch
+---
+
+While [Accumulo's installation instructions][install] are simple, it can be time consuming to install Accumulo
+given its requirement on [Hadoop] and [Zookeeper] being installed and running. For a one-time production
+installation, this set up time (which can take up to an hour) is not much of an inconvenience. However, it can become a burden
+for developers who need to frequently set up Accumulo to test code changes, switch between different
+versions, or start a fresh instance on a laptop.
+
+[Uno] and [Muchos] are tools that were created to ease this burden on developers by automating the installation
+of Accumulo and its dependencies. The names of Uno and Muchos indicate their use case. Uno is designed for running
+Accumulo on a single node while Muchos is designed for running Accumulo on a cluster of nodes. While Uno and
+Muchos are configured by default to install the most recent stable binary tarball release of Accumulo, Hadoop,
+and Zookeeper from Apache mirrors, they can easily be configured to use different versions to match a production
+cluster.
+
+The sections below provide brief tutorials on how to use these tools. For more complete documentation, view their
+respective GitHub pages.
+
+## Uno
+
+[Uno] is a command line tool that sets up Accumulo on a single machine for development. It can be installed by
+cloning the Uno git repo.
+
+```bash
+git clone https://github.com/astralway/uno.git
+cd uno
+```
+
+Uno works out of the box but it can be customized by modifying `conf/uno.conf`.
+
+Before setting up Accumulo, download the Accumulo, Hadoop, and Zookeeper tarballs from Apache by using the
```

Comment:
This could be even shorter

    First  download Accumulo, Hadoop, and Zookeeper using the command below: